contentcuration/contentcuration/frontend/shared/views/contentNodeFields/LearningActivityOptions.vue

```diff
@@ -1,47 +1,60 @@
 <template>
 
-  <DropdownWrapper>
-    <template #default="{ attach, menuProps }">
-      <VSelect
-        v-model="learningActivity"
-        :items="learningActivities"
-        :disabled="disabled"
-        box
-        chips
-        clearable
-        :label="translateMetadataString('learningActivity')"
-        multiple
-        deletableChips
-        :menu-props="menuProps"
-        :rules="learningActivityRules"
-        :attach="attach"
-      />
-    </template>
-  </DropdownWrapper>
+  <div>
+    <slot name="prependOptions"></slot>
+    <ExpandableSelect
+      v-model="learningActivity"
+      multiple
+      :disabled="disabled"
+      :expanded="expanded"
+      :options="learningActivities"
+      :hideLabel="hideLabel"
+      :rules="learningActivityRules"
+      :label="translateMetadataString('learningActivity')"
+    />
+  </div>
 
 </template>
 
 <script>
 
   import camelCase from 'lodash/camelCase';
   import { LearningActivities } from 'shared/constants';
+  import ExpandableSelect from 'shared/views/form/ExpandableSelect';
   import { constantsTranslationMixin, metadataTranslationMixin } from 'shared/mixins';
   import { getLearningActivityValidators, translateValidator } from 'shared/utils/validation';
-  import DropdownWrapper from 'shared/views/form/DropdownWrapper';
 
   export default {
     name: 'LearningActivityOptions',
-    components: { DropdownWrapper },
+    components: { ExpandableSelect },
     mixins: [constantsTranslationMixin, metadataTranslationMixin],
     props: {
+      /**
+       * It can receive a value as an array of strings of the selected learning activities, or
+       * an object with the following structure:
+       * {
+       *  [learningActivityId]: true | [nodeId1, nodeId2, ...]
```

Comment:
Would it simplify things to always require an Array of nodeIds here? In the single node case, it would just be a single nodeId.